src/main/proto/service.proto

```diff
@@ -17,35 +20,59 @@ service FeatureExtractor {
   rpc Graphlet (GraphletRequest) returns (FeaturesReply) {}
 }
 
-// The identifiers request message containing extractor configuration and uast.
-message IdentifiersRequest {
-  gopkg.in.bblfsh.sdk.v1.uast.Node uast = 1;
+message IdentifiersOptions {
   int32 docfreqThreshold = 2;
   int32 weight = 3;
   bool splitStem = 4;
 }
 
-// The literals request message containing extractor configuration and uast.
-message LiteralsRequest {
-  gopkg.in.bblfsh.sdk.v1.uast.Node uast = 1;
+message LiteralsOptions {
   int32 docfreqThreshold = 2;
   int32 weight = 3;
 }
 
-// The uast2seq request message containing extractor configuration and uast.
-message Uast2seqRequest {
-  gopkg.in.bblfsh.sdk.v1.uast.Node uast = 1;
+message Uast2seqOptions {
   int32 docfreqThreshold = 2;
   int32 weight = 3;
   int32 stride = 4;
   repeated int32 seqLen = 5;
 }
 
+message GraphletOptions {
+  int32 docfreqThreshold = 2;
```

Comment:
I guess it's ok to skip `1` and keep the same order as when this was in `GraphletRequest`, but is there a reason to do it?

Reply:
nope, no reason. I missed it because to copy-past. Thanks!

Reply:
fixed